jenkinsfile-runner-steward-image/scripts/run.sh

```diff
@@ -162,6 +162,29 @@ function with_termination_log() {
   return "$rc"
 }
 
+function with_retries() {
+  local retry_interval=$1 max_retries=$2
+  local cmd=("${@:3}")
+
+  local retry=1
+  local splitter_line=$(printf "%-50s" "*")
+  echo "Try $retry out of $max_retries max retries..."
+  "${cmd[@]}"
+  local rc="$?"
```

Comment:
If the command fails, the shell will exit immediately due to shell option `errexit` being used (by intention).

The non-zero exit code must be handled explicitly:

```suggestion
  local rc
  rc=0; "${cmd[@]}" || rc=$?
```